Documentation/kubernetes/policy.rst

```diff
@@ -31,7 +31,13 @@ with Kubernetes:
 
 It is recommended to only use one of the above policy types at a time to
 minimize unintended effects arising from the interaction between the
-policies.
+policies (there is no technical reason behind this recommendation, more 
+an organizational issue of managing unexpected combination of policy rules, when 
+policies of different types are applied. If you are understand difference 
+between those different policy types and managing policy rules accordingly 
+and have tools in place, like hubble, to troubleshoot unintended drops).
+
+
```

Comment:
Maybe we can reword to make this a bit more direct instead, eg:

> Cilium supports running multiple of these policy types at the same time. However caution should be applied when using multiple policy types at the same time, as it can be confusing to understand the complete set of allowed traffic across multiple policy types. If close attention is not applied this may lead to unintended policy allow behaviour. 

?

Reply:
@joestringer good suggestion, I've updated the PR